samples/orleans/OrleansAppHost/Program.cs

```diff
@@ -0,0 +1,36 @@
+using Microsoft.Extensions.Hosting;
+
+var builder = DistributedApplication.CreateBuilder(args);
+
+var storage = builder.AddAzureStorage("storage");
+
+if (builder.Environment.IsDevelopment())
```

Comment:
Suggest something like:
```csharp
if (!args.Contains("manifest")) // AZD UP passes in --publisher manifest
```
Since `AppHost` only exists in development, differentiating on `Environment` does not work.
The intention is to differentiate between running the application locally versus generating an Aspire deployment manifest; the latter is done by running the Apphost with command-line parameter `--publisher manifest`

Reply:
thanks for this comment, this is relay important to know, about is it in infrastruction or in debug mode on local machine